common.mk

```diff
@@ -10,6 +10,7 @@ MASTER_CTR:= dcos-docker-master
 AGENT_CTR := dcos-docker-agent
 PUBLIC_AGENT_CTR := dcos-docker-pubagent
 INSTALLER_CTR := dcos-docker-installer
+INSTALLER_PORT := 9000
```

Comment:
i wonder if this would better go in the Makefile... i have no idea why each thing is in each file... seems like things that users can override should go int he makefile, but that doesn't seem like the existing pattern